Seminar_005/Homework_005/Simple_028.py

```diff
@@ -0,0 +1,78 @@
+# Задача 28:
+# Напишите рекурсивную функцию sum(a, b),
+# возвращающую сумму двух целых неотрицательных чисел.
+# Из всех арифметических операций допускаются только +1 и -1.
+# Также нельзя использовать циклы.
+
+# *Пример:*
+
+# 2 2
+#     4
+
+
+# variant 1
+print("Вариант 1")
+def sum(a, b):
+    if b == 0:
+        return a
+    return sum(a+1, b-1)
+
+
+a = int(input("Введите число А = "))
+b = int(input("Введите числов B = "))
+
+print(f'Сумма чисел {a} и {b} равна {sum(a, b)}')
+
+# variant 2
+print("-----------------------------")
+print("Вариант 2")
+def sum(a,b):
+    if a<b:
+        a,b = b,a
+    if b==0:
+        return a
+    elif a==0:
+        return b
+    else:
+        return sum(a+1,b-1)
```

Comment:
Не забывайте про необходимые пробелы
```suggestion
def sum(a,b):
    if a < b:
        a, b = b, a
    if b == 0:
        return a
    elif a == 0:
        return b
    else:
        return sum(a + 1, b - 1)
```